mongo/datadog_checks/mongo/mongo.py

```diff
@@ -171,80 +156,20 @@ def __init__(self, name, init_config, instances=None):
         self.collections_indexes_stats = is_affirmative(self.instance.get('collections_indexes_stats'))
         self.coll_names = self.instance.get('collections', [])
         self.custom_queries = self.instance.get("custom_queries", [])
+        # By default consider that this instance is a standalone, updated on each check run.
+        self.deployment_type = StandaloneDeployment()
```

Comment:
(Nit) We must rename this attribute

```suggestion
        self.deployment = StandaloneDeployment()
```